editor/src/clj/editor/collection_data.clj

```diff
@@ -1,33 +1,361 @@
+;; Copyright 2020-2022 The Defold Foundation
+;; Copyright 2014-2020 King
+;; Copyright 2009-2014 Ragnar Svensson, Christian Murray
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
 (ns editor.collection-data
-  (:import [java.io StringReader]))
```

Comment:
We used to have a file named `collection_data.clj` with utility functions convert embedded strings in collections and game objects into maps. I've renamed that file `collection_string_data.clj`, so that's why this diff looks like I've removed a bunch of functions.